packages/ember-metal/lib/computed.js

```diff
@@ -257,6 +257,12 @@ ComputedPropertyPrototype.property = function() {
         `Please refactor from \`Ember.computed('${property}', function() {});\` to \`Ember.computed('${property.slice(0, -6)}.[]', function() {})\`.`,
       property.slice(-5) !== '@each'
     );
+    warn(
```

Comment:
should it not be an assertion? also, should we say the workaround is to create an intermediary computed property?